source/include/FreeRTOS_IP_Common.h

```diff
@@ -47,6 +47,14 @@ typedef union IP_Address
     IPv6_Address_t xIP_IPv6; /**< IPv6 address */
 } IP_Address_t;
 
+/** @brief A struct that can hold either an IPv4 or an IPv6 address. */
+typedef struct xxIPv46_Address
+{
+    /* A struct that can hold either an IPv4 or an IPv6 address. */
+    IP_Address_t xIPAddress; /**< The IPv6-address. */
```

Comment:
`xIPAddress` can be used to hold either v4 or v6 address here, comment only says about v6.

Reply:
We already have a structure which holds both v4 and v6 address, why is this needed?

Reply:
Not sure why we are introducing this structure.